docs/sections/user_guide/cli/tools/mode_template.rst

```diff
@@ -136,35 +136,6 @@ and a YAML file called ``values.yaml`` with the following contents:
      $ uw template render --input-file template --values-file values.txt --values-format yaml
      Hello, World!
 
-* It is an error to render a template without providing all needed values. For example, with ``recipient: World`` removed from ``values.yaml``:
```

Comment:
This block moved down in the doc, under a **NB** heading that makes it more obvious that several tests that follow use a modified `values.yaml` file with a missing value.